MonoGame.Framework/Utilities/OS.cs

```diff
@@ -0,0 +1,46 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace MonoGame.Utilities
+{
+    /// <summary>
+    /// Defines the operating system.
+    /// </summary>
+    public enum OS
```

Comment:
> Isn't the entire argument of DesktopGL that people shouldn't be writing platform specific code?
> 
> They should be using #if PLATFORM_WINDOWS and the equivalent for other platforms if they want to write platform-specific code.

People might still want to be able to check current OS from code. For example I am working on a game where when an error occurs I made it load fake "Blue screen of death" if on Windows, or "Kernel panic" if on Unix. Yes, I could have used "#if PLATFORM_WINDOWS" but then my .exe wouldn't be cross platform.